build.gradle

```diff
@@ -74,21 +74,24 @@ configurations {
 }
 
 dependencies {
-    compile 'com.datastax.cassandra:cassandra-driver-core:2.1.8'
+    compile 'com.datastax.cassandra:cassandra-driver-core:3.1.0'
     compile 'org.slf4j:slf4j-api:1.7.12'
     compile 'com.google.code.findbugs:jsr305:3.0.0'
 
     testCompile 'org.apache.logging.log4j:log4j-core:2.2'
     testCompile 'junit:junit:4.12'
     testCompile 'org.assertj:assertj-core:2.0.0'
+    testCompile 'org.assertj:assertj-joda-time:2.0.0'
+    testCompile 'joda-time:joda-time:2.9.4'
     testCompile 'org.hamcrest:hamcrest-all:1.3'
     testCompile 'org.mockito:mockito-core:1.10.19'
     testCompile 'org.powermock:powermock-api-mockito:1.6.5'
     testCompile 'org.powermock:powermock-module-junit4:1.6.5'
 
     testCompile 'org.scassandra:java-client:1.0.3'
 
-    testCompileAndFunctional 'org.cassandraunit:cassandra-unit-shaded:2.1.9.2'
+//    testCompileAndFunctional 'org.cassandraunit:cassandra-unit-shaded:2.1.9.2'
```

Comment:
If this is staying commented out then it should be removed, though I like @oliverlockwood's idea of testing against multiple versions.